frame_test.go

```diff
@@ -51,7 +51,9 @@ func TestFrameConn(t *testing.T) {
 							return
 						}
 					}
-					fmt.Println(string(b))
+					if len(string(b)) > 3 {
```

Comment:
In the test, it seem to print a lot of redundant change lines, so I added this to make test look a bit nicer.